web/source/osk/embedded/subkeyDelegator.ts

```diff
@@ -0,0 +1,87 @@
+/// <reference path="../realizedGesture.interface.ts" />
+
+namespace com.keyman.osk.embedded {
+  // "Delegator", rather than "Popup", because KMW delegates display + selection
+  // of subkeys to the host app when in the embedded context.
```

Comment:
Can we turn this into a formal comment following our Brand Spanking New style guide? https://github.com/keymanapp/keyman/wiki/Keyman-Code-Style-Guide